src/home/homectl.c

```diff
@@ -3485,6 +3488,17 @@ static int parse_argv(int argc, char *argv[]) {
                         strv_uniq(arg_pkcs11_token_uri);
                         break;
 
+                case ARG_FIDO2_CRED_ALG:
+                        if (streq(optarg, "ES256"))
+                                arg_fido2_cred_alg = COSE_ALG_ES256;
+                        else if(streq(optarg, "RS256"))
+                                arg_fido2_cred_alg = COSE_ALG_RS256;
+                        else if(streq(optarg, "EDDSA"))
+                                arg_fido2_cred_alg = COSE_ALG_EDDSA;
+                        else
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Unsupported COSE algorithm: %s", optarg);
```

Comment:
How about to introduce a simple helper `parse_fido2_algorithm()`?

I am not familiar with each algorithm, but why `COSE_ECDH_ES256` is excluded here?

Reply:
> How about to introduce a simple helper `parse_fido2_algorithm()`?
> 
> I am not familiar with each algorithm, but why `COSE_ECDH_ES256` is excluded here?

Ok I'll consider introduce the helper.
As libfido2 manpage `man fido_cred_set_type` state, it only accepts the three values.
Source code:https://github.com/Yubico/libfido2/blob/main/src/cred.c

Reply:
I see. Thanks.